fluent-plugin-google-cloud.gemspec

```diff
@@ -19,7 +19,7 @@ eos
   gem.test_files    = gem.files.grep(/^(test)/)
   gem.require_paths = ['lib']
 
-  gem.add_runtime_dependency 'fluentd', '1.12.3'
+  gem.add_runtime_dependency 'fluentd', '1.11.2'
```

Comment:
We also need to update https://github.com/GoogleCloudPlatform/google-fluentd/blob/master/config/software/fluentd.rb

Reply:
Done in https://github.com/GoogleCloudPlatform/google-fluentd/pull/340. Also added a note for the two locations to update.